store/src/java/com/zimbra/cs/util/ProxyConfGen.java

```diff
@@ -1756,6 +1756,30 @@ public void update() throws ServiceException {
     }
 }
 
+/**
+ * Simplified container object for a server
+ * @author Davide Baldo
```

Comment:
Nit: remove @author tag - going forward, it would seem perhaps better if we stop using/relying on @author and just use git history. See also:
- https://stackoverflow.com/questions/17269843/javadoc-author-tag-good-practices
- https://www.vojtechruzicka.com/stop-using-javadoc-author-tag/